calico/about/product-comparison.mdx

```diff
@@ -0,0 +1,87 @@
+---
+description: Describes Tigera products, including a feature comparison table.
+---
+
+# Product comparison
+
+**{{prodname}} Open Source** is the base product that comprises both {{prodname}} Enterprise and {{prodname}} Cloud. It provides the core networking and network policy features.
+
+**{{prodname}} Enterprise** adds a web UI for visibility and troubleshooting Kubernetes clusters, and other value-added features on top of {{prodname}} Open Source. 
```

Comment:
I feel like this sells Enterprise short, and makes it seem more like it's just a UI on top of OSS Calico, which isn't true - there are a lot of advanced features. If anything, the UI is the least important part of enterprise (it's more relevant to CC)

Reply:
Good point. Rewritten.